AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts

```diff
@@ -349,6 +376,19 @@ export class AssignJudgeComponent extends BookingBaseComponent implements OnInit
         this.expanded = !this.expanded;
     }
 
+    validateJudgeAndJohMembers(): boolean {
+        if (this.hearing?.participants.length > 0 && this.judgeName.value) {
+            const johMembers = this.hearing.participants.filter(
+                x => x.hearing_role_name === 'Panel Member' || x.hearing_role_name === 'Winger'
+            );
+            if (johMembers?.length > 0) {
```

Comment:
could use `.some()` to check if its empty or not

Reply:
Done